### PR TITLE
chore: arm compatible test tools

### PR DESCRIPTION
**Pull request by bwagner5**
Fixes # 

**Description**

**How was this change tested?**

* built on arm64 instance

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/tools/Dockerfile [hunk 1]
```diff
@@ -45,7 +45,7 @@ COPY --from=installer /aws-cli-bin/ /usr/local/bin/
 RUN aws --version
 
 # Install eksctl
```
Comment (jonathan-innis) [inline test/tools/Dockerfile:47]: Why do we need this?

Comment (bwagner5) [reply test/tools/Dockerfile:47]: because I tried to test this on my arm instance :D

Comment (jonathan-innis) [reply test/tools/Dockerfile:47]: We are running this on ARM instances?

Comment (jonathan-innis) [reply test/tools/Dockerfile:47]: Don't need to block on this, just curious

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Review by ellistarn — APPROVED** [review]

File: test/tools/Dockerfile (hunk 1 above)
Comment (bwagner5) [reply test/tools/Dockerfile:47]: we're not running this on ARM instances, but wanted to test it and the instance I was using happened to be ARM. Trivial change to make it ARM compatible so figured I'd PR it. We may want to run it on ARM instances at some point

**Pull request merged**